puppet/zulip/manifests/postgres_common.pp

```diff
@@ -7,8 +7,6 @@
                         "ptop",
                         # Python modules used in our monitoring/worker threads
                         "python-gevent",
-                        "python-tz",
-                        "python-dateutil",
```

Comment:
Let's just leave these dependencies installed, and open an issue for cleaning this up later.  I'll have a good test environment for this available in a few weeks and can deal with it then